lib/beetle/configuration.rb

```diff
@@ -35,6 +35,12 @@ class Configuration
     # handler timeout.
     attr_accessor :redis_failover_timeout
 
+    # how long we want status keys survive after we are we have seen the second message of
+    # a redundant message pair. Setting this to a high value (hours) will reduce the
+    # likelyhood of executing handler logic, but require a lerger redis instance and cause
```

Comment:
`s/likelyhood/likelihood`

Reply:
`s/lerger/larger`

Reply:
What does "larger redis instance" mean? I think we only mean larger in memory, so we should be clear.

Reply:
fixed